src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -46,6 +46,8 @@ export class TechRecordTitleComponent implements OnInit {
   }
 
   navigateToArchive(): void {
-    this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/${this.vehicleTechRecord?.vin}/archive`);
+    this.router.url.endsWith('provisional')
```

Comment:
I thought there was PR to change the provisional check to checking the last two parts of the URL. Not saying this is wrong in this context, but would that scenario not be applicable here?

Reply:
https://github.com/dvsa/cvs-app-vtm/pull/666 was the PR. Also ideally, we should only have one function/method/selector/way of knowing if a record is provisional. In the tech record service would be ideal I guess.

Reply:
Check if you like that now